### ISSUE TITLE
Xrefs are allowed in shortdesc

### ISSUE THREAD

**Issue by raducoravu** [hyperwrite/DITAStyleGuide#7]
See: https://www.oxygenxml.com/forum/post66722.html#p66722

**Comment by hyperwrite** [comment]
This is a tricky one, because the Style Guide is really only up-to-date for DITA 1.2.

**Comment by raducoravu** [comment]
Right, I understand. In general it's not a great idea to add a link in a shortdesc because the shortdesc may be used as a tooltip and tooltips contain plain text, mapping a link to plain text is not straight forward.

**Comment by hyperwrite** [comment]
I have added a note to c_Formatting_in_shortdesc.dita mentioning that this is allowed in 1.3, while not changing the general guidance that cross-references should not be used.
